grpc/sa-wrappers.go

```diff
@@ -671,11 +672,8 @@ func (sas StorageAuthorityServerWrapper) FQDNSetExists(ctx context.Context, requ
 	return &sapb.Exists{Exists: &exists}, nil
 }
 
-func (sac StorageAuthorityServerWrapper) PreviousCertificateExists(
-	ctx context.Context,
-	req *sapb.PreviousCertificateExistsRequest,
-) (*sapb.Exists, error) {
-	if req == nil || req.Domain == nil || req.RegID == nil {
```

Comment:
I think Domain and RegID should always be nonzero in these calls (and therefore the nil checks can stay until we migrate this proto).